flask_itsyouonline.py

```diff
@@ -257,3 +259,5 @@ def _wrapper(*args, **kwargs):
         return _wrapper
     return decorator
 
+def is_member_of(org, scopes):
```

Comment:
I would expect this method to use the session object to get the scopes of the current user

Reply:
Or maybe add a helper 

def is_current_user_member_of(org):
      pass